### PR TITLE
 Calculations and run plan for OAV loop  centring

### PR DESCRIPTION
**Pull request by evalott100**
Fixes #420 
Replaces #453, since that branch contained other changes.
To test:
* Run against s03 and the branch of issue 26 in 
* Check output is acceptable.

### CODE REVIEW

**Review by DominicOram — CHANGES_REQUESTED** [review]
This is a review from readability more than correctness, will test correctness more after lunch. As discussed I think that we should label the co-ordinate system coming out of the OAV as i, j, k and be explicit somewhere how that is defined relative to x, y, z.

Should: Do we have the plan we tested on the beamline that moves to tip centre? We should recover it and put it in a PR

Should: I think coming up with some better datastructures would be good, rather than just numpy arrays everywhere. I'll write an issue for this

File: src/artemis/devices/oav/oav_calculations.py [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+        omega_angles[index_orthogonal_to_largest_width]
+    )
+
+    # Store the y value which will be the magnitude in the z axis on 90 degree rotation
+    z_pixels = int(y_positions[index_orthogonal_to_largest_width])
+
+    #
+    if widest_omega_angle_orthogonal is None:
+        LOGGER.error("Unable to find loop at 2 orthogonal angles")
+        return
```
[the remark this answers is not in the document]
Comment (evalott100) [reply src/artemis/devices/oav/oav_calculations.py:218]: You're right it's an unnecessary check, an oversight from before I split into smaller methods.

**Review by DominicOram — CHANGES_REQUESTED** [review]
This isn't going to work in it's current state and it needs further testing on the beamline

**Review by DominicOram — APPROVED** [review]
This was successfully tested on the beamline and so I'm happy to merge it

**Pull request merged**